index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>How2Car</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/4.2.0/normalize.min.css">
```

Comment:
You can leave the protocol off of the URL to your CDN links, such as: 
``` html
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/normalize/4.2.0/normalize.min.css">
```

This will let the deployment machine determine the best protocol to use and apply it.

Note: If you're not working on a live-server, node server, etc, this will not work for you. It'll try and open the default protocol on your machine, which is `file://`.